e2e-tests/e2e-fixtures/mock-gef-facilities.js

```diff
@@ -186,6 +186,8 @@ const anUnissuedContingentFacility = ({ facilityEndDateEnabled = false } = {}) =
   canResubmitIssuedFacilities: null,
 });
 
+exports.anUnissuedContingentFacility = anUnissuedContingentFacility;
```

Comment:
I'd prefer to see these exports moved down to the bottom of the file (inc the existing ones), as I feel they're slightly hidden in the code here -- What do you think? (also cc @ttbarnes in case this is something he prefers)

Reply:
TBH i'm not too fussed, we want to ultimately replace things like this with `export const`